src/assets/translations/en/main.json

```diff
@@ -600,7 +600,8 @@
         "header": "Confirm Deposit",
         "averageApr": "Average APR",
         "estimatedReturns": "Estimated Yearly Returns*",
-        "preFooter": "Rewards accrue automatically"
+        "preFooter": "Rewards accrue automatically",
+        "notEnoughGas": "Not enough ETH for gas"
```

Comment:
Non-blocking, but this would be more future-proof with some string interpolation, see this translation as an example: 
https://github.com/shapeshift/web/blob/cc75427517e7c0eef5b6362a30a0286c04ced237/src/assets/translations/en/main.json#L514

The reason for that is, as we start supporting more EVM chains, the native token might be different than ETH then